SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialActorUtils.h

```diff
@@ -34,6 +34,12 @@ inline AActor* GetHierarchyRoot(const AActor* Actor)
 	return Owner;
 }
 
+inline AActor* GetHierarchyRoot(const AActor* Actor)
+{
+	AActor* TopmostOwner = GetTopmostOwner(Actor);
+	return TopmostOwner ? TopmostOwner : const_cast<AActor*>(Actor);
```

Comment:
Is this const casting because the locking policy has the wrong signature? Or do we need to get non const actors from const actors?

Reply:
GetOwner gives you a non const actor pointer.
And from the POV of this function, the given actor is not modified.